src/eventshub/src/component/SortBy/SortBy.tsx

```diff
@@ -41,10 +42,10 @@ export const SortBy = ({ activeTab }: SortByProps) => {
   return (
     <div className="er-flex er-direction-row er-items-center er-w-fit er-ml-auto">
       <label htmlFor={id} className="er-m-0 er-max-w-fit er-w-full er-text-16 er-h-fit">Sort by: </label>
-      <select id={id} onChange={handleChange} value={activeSort} className="er-m-0 er-py-0 er-pl-6 er-pr-36 er-max-w-fit er-w-full er-text-16">
-        <option value='relevance'>Relevance</option>
-        <option value='asc'>Name: A-Z</option>
-        <option value='desc'>Name: Z-A</option>
+      <select id={id} value={activeSort} onChange={handleChange} className="er-m-0 er-py-0 er-pl-6 er-pr-36 er-max-w-fit er-w-full er-text-16">
```

Comment:
having the value set here like this is supposed to have the options `selected` in the rendered markup but for the life of me I can't figure out why it is able to set the element correctly visually, and has selected=true in the property inspector, but doesn't have the attribute on the HTML element!